ichnaea/tests/base.py

```diff
@@ -299,6 +300,10 @@ def cleanup_tables(cls, engine):
 
 class RedisIsolation(object):
 
+    def tearDown(self):
+        self.redis_client.flushdb()
+        super(RedisIsolation, self).tearDown()
+
```

Comment:
I don't understand your reasoning here.  If you include the RedisIsolation mixin in a test case, you want to be able to access the redis client inside your tests.  If you have tests which might interact with a redis database, you want to flush that database after each test is run.  Where should the code for flushing the db live?